SNspread/.Rhistory

```diff
@@ -0,0 +1,512 @@
+n_friendships = sample(1:n, 1)
+edgelist2 = data.frame(
+input = friends[sample(1:n, n_friendships)],
+output = friends[sample(1:n, n_friendships)],
+connection = rep(1,n_friendships)
+)
+# Complete edgelist
+edgelist = rbind(edgelist, edgelist2)
+# plot graph
+qgraph::qgraph(edgelist, color = c("red","blue"))
+# Use colors for spread, create new graph every 5 seconds
+}
+create_graph(n=20)
+create_graph <- function (n = 10){# n is number of friends
+# Make edgelist
+friends = paste("friend", 1:n, sep = "")
+edgelist = data.frame(
+input = rep("user", n),
+output = friends,
+connection = rep(1,n))
+# Simulate some additional friendships
+n_friendships = sample(1:n, 1)
+edgelist2 = data.frame(
+input = friends[sample(1:n, n_friendships)],
+output = friends[sample(1:n, n_friendships)],
+connection = rep(1,n_friendships)
+)
+# Complete edgelist
+edgelist = rbind(edgelist, edgelist2)
+# plot graph
+qgraph::qgraph(edgelist, color = c(rep("red",3), "blue"))
+# Use colors for spread, create new graph every 5 seconds
+}
+create_graph(n=20)
+create_graph <- function (n = 10){# n is number of friends
+# Make edgelist
+friends = paste("friend", 1:n, sep = "")
+edgelist = data.frame(
+input = rep("user", n),
+output = friends,
+connection = rep(1,n))
+# Simulate some additional friendships
+n_friendships = sample(1:n, 1)
+edgelist2 = data.frame(
+input = friends[sample(1:n, n_friendships)],
+output = friends[sample(1:n, n_friendships)],
+connection = rep(1,n_friendships)
+)
+# Complete edgelist
+edgelist = rbind(edgelist, edgelist2)
+# plot graph
+qgraph::qgraph(edgelist, color = c(rep("red",3), "blue"))
+# Use colors for spread, create new graph every 5 seconds
+qgraph::qgraph(edgelist, color = c(rep("red",2), "blue"))
+qgraph::qgraph(edgelist, color = c("red", "blue"))
+qgraph::qgraph(edgelist, color = c("red"))
+}
+create_graph(n=15)
+# Simualte data!
+library(tidyr)
+library(lattice)
+library(ggplot2)
+sd_CI <- function (n, mean, cilow) {
+var = sqrt(n)*((-cilow[1] + mean)/1.96)
+sd = sqrt(var)
+return(sd)
+}
+# t0 (intercept)
+nrt = data.frame(subject = factor(1:126))
+nrt$t0 = rnorm(126, 16.44, sd = 10.5)
+nrt$t1.5 = rnorm(126, 18.61, sd = 12.2)
+nrt$t3 = rnorm(126, 16.38, sd = 10.9)
+nrt_long = gather(nrt, timepoint, IDSscore, t0:t3, factor_key = T)
+contr = data.frame(subject = factor(127:239))
+contr = data.frame(subject = factor(127:239))
+contr$t0 = rnorm(113, 16.06, sd = 9.5)
+contr$t1.5 = rnorm(113, 20.76, sd = 12.1)
+contr$t3 = rnorm(113, 21.52, sd = 12.14)
+contr_long = gather(contr, timepoint, IDSscore, t0:t3, factor_key = T)
+contr_long$condition = 0
+complete = rbind(nrt_long, contr_long)
+View(contr_long)
+nrt_long$condition = 1
+complete = rbind(nrt_long, contr_long)
+complete$time = numeric(nrow(complete))
+for (i in 1:nrow(complete)) {
+if (complete$timepoint[i] == "t0") {
+complete$time[i] = 0
+} else if (complete$timepoint[i] == "t1.5") {
+complete$time[i] = 1.5
+} else if (complete$timepoint[i] == "t3") {
+complete$time[i] = 3
+}
+}
```

Comment:
Maybe you could revisit the google style guide to make some of this a little more understandable :) The one from tidyverse is also good: https://style.tidyverse.org/syntax.html#function-calls